### PR TITLE
[Feature]: Add new  update header component

### PR DESCRIPTION
**Pull request by Aakashrawat08**
## Description

-   [Ticket Link](#) (if any)
-   Provide a small description ( 2 - 3 lines) of the Pull Request

## UI screenshot or Video

-   Please include a screenshot or video to help reviewers understand the specific part of the application being modified. Multiple screenshots are appreciated

---

### General:

-   [x] **PR Size :** Confirming PR size < 20 files or pre-approved if > 20 files.
-   [x] **Self Review :** Confirming I have self-reviewed my PR
-   [x] **Peers Review :** - [ ] @Dev 1
-   [x] Confirming I have added all sensitive information-related should be inside .env files

### Client:

-   [x] **Console :** Confirming no console errors in the UI for this update.

-   [x] **Import order :** Confirming correct import order.

```
# External Import
import React, { useState, useEffect } from 'react';
import { useParams } from 'react-router-dom';
import { useDispatch, useSelector, shallowEqual } from 'react-redux';

# Internal Import
import { } from '../../Api/ApiRoutes';
import { } from '../../constants/AppConstant';
```

-   [x] **Re-renders :** Confirming minimal re-renders for new React components.

-   [x] **Network calls :** Confirming efficient network/API calls, minimized as needed.

-   [x] **Util function comments :** Confirming comprehensive comments for utility functions.

-   [x] **Image optimization :** Confirming I have checked that all images are optimized. If not, please use https://tinyjpg.com/ or https://vecta.io/nano.

-   [x] **Dead code :** Confirming I have removed all comments code which is of no use OR written why I have kept the commented code.

### Confirming different viewport testing of the entire flow:

-   [x] Tested on 280px viewport
-   [x] Tested on 320px viewport
-   [x] Tested on 375px viewport
-   [x] Tested on 425px viewport
-   [x] Tested on 768px viewport
-   [x] Tested on 1024px viewport
-   [x] Tested on 1440px viewport
-   [x] Tested on 2560px viewport

### End-to-end testing:

-   [x] **Happy flow :** Successfully tested end-to-end happy flow, all working as expected.
-   [x] **Error flow :** Tested UI flow with error scenarios, gracefully handled errors on UI
-   [ ] **Common component changed :** Tested change in common component across views to prevent breaking elsewhere.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -2,14 +2,79 @@
...
+                '#828796': '#828796',
+                '#2F3750': '#2F3750'
+            },
+            borderRadius: {
+                8: '2rem'
+            },
+            fontSize: {
+                3: '0.875rem',
+                7: '1.75rem',
+                4: '1rem',
```
Comment (Kuldeep-primathon) [inline tailwind.config.js]: 1 rem is already available.
class - text-base

File: tailwind.config.js [hunk 2]
```diff
@@ -2,14 +2,79 @@
...
+                '#0A66EC': '#0A66EC',
+                '#828796': '#828796',
+                '#2F3750': '#2F3750'
+            },
+            borderRadius: {
+                8: '2rem'
+            },
+            fontSize: {
+                3: '0.875rem',
+                7: '1.75rem',
```
Comment (Kuldeep-primathon) [inline tailwind.config.js]: 0.875rem -> text-sm

instead of 7: '1.75rem'  , user '2.5xl': '1.75rem'

File: tailwind.config.js [hunk 3]
```diff
@@ -2,14 +2,79 @@
...
+                4: '1rem',
+                4.5: '1.125rem',
```
Comment (Kuldeep-primathon) [inline tailwind.config.js]: it is text-lg

File: tailwind.config.js [hunk 4]
```diff
@@ -2,14 +2,79 @@
...
+                5: '1.25rem'
```
Comment (Kuldeep-primathon) [inline tailwind.config.js]: it is text-xl

**Comment by Aakashrawat08** [comment]
https://github.com/primathonacademy/august-batch-live-project/assets/87071775/4f0a7570-fce7-4c65-a77a-00c25e0177c2

**Review by Kuldeep-primathon — APPROVED** [review]

**Review by Kuldeep-primathon — APPROVED** [review]

**Pull request merged**